Source/Core/Core/NetPlayServer.cpp

```diff
@@ -981,16 +976,18 @@ void NetPlayServer::unmapPortThread()
 // discovers the IGD
 bool NetPlayServer::initUPnP()
 {
+  static bool s_inited = false, s_error = false;
+
   std::vector<UPNPDev*> igds;
```

Comment:
I can go one better and remove the vector entirely. I'll put this in #5898 with the other cleanups.